### PR TITLE
Some grammar/spelling errors.

### PR DESCRIPTION
**Pull request by actuallyarm64**
im currently reading the whole documentation. Please do not merge until ALL spelling or grammar errors have been fixed. Thank you.

### CODE REVIEW

File: src/docs/unbricking/index.md [hunk 1]
```diff
@@ -33,6 +33,7 @@ While this is somewhat device-specific, the main points are the same:
 
 * Disconnect all external power
 * Remove bottom cover (screws are often located under rubber feet or strips)
+        - In some Chromebooks, you eill need to remove the keyboard, and as mentioned in [Disabling write protect via Battery](/firmware/battery.html), theres a ribbon wire thats needs to be CAREFULLY removed.
```
Comment (meghan06) [inline src/docs/unbricking/index.md:36]: "Will" is spelled wrong. 

This sentence should be reworded. Current wording makes it sound like all Chromebooks have this cable.

File: src/docs/unbricking/index.md [hunk 2]
```diff
@@ -167,6 +168,7 @@ As above, this is only needed if you failed to factory reset the CCD flags as pa
...
```
Comment (meghan06) [inline src/docs/unbricking/index.md:171]: Typo. 

See previous comment too

**Review by meghan06 — CHANGES_REQUESTED** [review]
You should distinguish that some CBs open up via back and some via keyboard. Also read comments.

File: src/docs/unbricking/index.md (hunk 2 above)
Comment (actuallyarm64) [reply src/docs/unbricking/index.md:171]: Im on it 👍

File: src/docs/unbricking/index.md [hunk 3]
```diff
@@ -33,6 +33,7 @@ While this is somewhat device-specific, the main points are the same:
 
 * Disconnect all external power
 * Remove bottom cover (screws are often located under rubber feet or strips)
+        - Some Chromebooks open up via back and some via keyboard, and as mentioned in [Disabling write protect via Battery](/firmware/battery.html). On keyboard, you have to pry it out and remove a ribbon wire under the keyboard.
```
[the remark this answers is not in the document]
Comment (ethanaobrien) [reply src/docs/unbricking/index.md:36]: Did you forget to push or something?

**Review by WeirdTreeThing — APPROVED** [review]

**Comment by actuallyarm64** [comment]
> Waiting on the changes I requested to be addressed

They are already there

**Review by ethanaobrien — APPROVED** [review]

**Review by WeirdTreeThing — APPROVED** [review]

**Pull request merged**